perf/load/common.sh

```diff
@@ -16,6 +16,7 @@ function run_test() {
 
    # remove stdio rules
   kubectl --namespace istio-system delete rules stdio stdiotcp || true
+  kubectl --namespace istio-ingress delete rules stdio stdiotcp || true
```

Comment:
This and the previous line is no longer needed. We don’t create stdio by default now